mybb/mybb
# Fixes #3995 Only parent forum's mod/modcp permissions are used in ModCP

Resolves #3995.

Test this PR:
1. Create forum A and its sub-forum B.
1. Assign a user U as moderator of forum A and B, explicitly. Make sure U or U's usergroup isn't any other forum's moderator.
1. Arbitrarily set different moderating permissions for the user U for forums A and B. For example, `canviewmodlog` is `yes` in forum A but `no` in forum B, `canmanageannouncements` is `no` in forum A but `yes` in forum B.
1. Go to ModCP's relevant sections to see if permissions are in effect.
